frontend/app/routes/_public+/apply+/$id+/federal-provincial-territorial-benefits.tsx

```diff
@@ -0,0 +1,112 @@
+
+import { Button, ButtonLink } from '~/components/buttons';
+import { InputRadios } from '~/components/input-radios';
+import { getApplyFlow } from '~/routes-flow/apply-flow';
+import { getLookupService } from '~/services/lookup-service.server';
+import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('provincial-territorial');
+
+export const handle = {
+  i18nNamespaces,
+  pageIdentifier: 'CDCP-1115',
+  pageTitleI18nKey: 'provincial-territorial:title',
+};
+
+export async function loader({ request, params }: LoaderFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id, state } = await applyFlow.loadState({ request, params });
+  const federalDentalBenefits = await getLookupService().getAllFederalDentalBenefit();
+  const federalSocialPrograms = await getLookupService().getAllFederalSocialPrograms();
+  return json({ id, state, federalDentalBenefits, federalSocialPrograms });
+}
+
+export async function action({ request, params }: ActionFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id } = await applyFlow.loadState({ request, params });
+
+  const formData = Object.fromEntries(await request.formData());
+  const parsedDataResult = applyFlow.dentalBenefitsStateSchema.safeParse(formData);
+
+  if (!parsedDataResult.success) {
+    return json({
+      errors: parsedDataResult.error.format(),
+      formData: formData as Partial<z.infer<typeof applyFlow.dentalBenefitsStateSchema>>,
+    });
+  }
+
+  const sessionResponseInit = await applyFlow.saveState({
+    request,
+    params,
+    state: { dentalBenefit: parsedDataResult.data },
+  });
+
+  return redirect(`/apply/${id}/confirm`, sessionResponseInit);
+}
+
+export default function AccessToDentalInsuranceQuestion() {
+  const { federalSocialPrograms, federalDentalBenefits, state } = useLoaderData<typeof loader>();
+  const { i18n, t } = useTranslation(i18nNamespaces);
+  const [checked, setChecked] = useState(state.dentalBenefit?.federalBenefit ?? '');
+
+  return (
+    <Form method="post">
+      <p className="mb-4">{t('provincial-territorial:access-to-dental')}</p>
+      <p>{t('provincial-territorial:eligibility-criteria')}</p>
+      {federalDentalBenefits.length > 0 && (
+        <div className="my-6">
+          {federalDentalBenefits.length > 0 && (
+            <InputRadios
+              id="federal-benefit"
+              name="federalBenefit"
+              legend={t('provincial-territorial:federal-benefits.legend')}
+              options={federalDentalBenefits.map((option) => ({
+                children: (
+                  <div>
+                    <strong>{getNameByLanguage(i18n.language, option)}</strong>,&nbsp;
+                    {option.code === 'yes' ? <Trans ns={i18nNamespaces} i18nKey="provincial-territorial:federal-benefits.option-yes" /> : <Trans ns={i18nNamespaces} i18nKey="provincial-territorial:federal-benefits.option-no" />}
+                  </div>
```

Comment:
FYI this page produces the following validation error when ran against an [HTML validator](https://validator.w3.org/):
"Error: Element [div](https://html.spec.whatwg.org/multipage/#the-div-element) not allowed as child of element [label](https://html.spec.whatwg.org/multipage/#the-label-element) in this context."

Maybe we should use a `<span>` or Fragment `<>` instead of a `<div>` here.

Reply:
Fragment <> instead of a <div> here like @nicholas-ly suggested.